MovieGallery/Pages/Search.cshtml

```diff
@@ -13,8 +13,8 @@
                 <div class="col-md-12">
                     <div class="search">
                         <i class="fa fa-search"></i>
-                        <input type="text" class="form-control" placeholder="Enter year to start search..." name="SearchbyYear" id="SearchbyYear">
-                        <button type="submit" class="btn btn-secondary">Search by Year</button>
+                        <input type="text" class="form-control" placeholder="Enter year to start search..." name="SearchbyYear" id="SearchbyYear" >
+                        <button type="submit" class="btn btn-secondary">Search</button>
```

Comment:
Instead of search by year in the button, I have added just search as the placeholder already mentions to enter the year to start the search. It is not a good practice to add search by year in a button description.

Reply:
This is the closest context to searching by year, so I think it makes the most sense here.